gdk4/src/lib.rs

```diff
@@ -65,3 +68,39 @@ pub use rectangle::Rectangle;
 pub use rgba::{RgbaParseError, RGBA};
 pub use surface::SurfaceExtManual;
 pub use time_coord::TimeCoord;
+
+// This is the priority that the idle handler processing surface updates is given in the glib::MainLoop.
+#[doc(alias = "GDK_PRIORITY_REDRAW")]
+pub const PRIORITY_REDRAW: u32 = ffi::GDK_PRIORITY_REDRAW as u32;
+
+// A mask covering all entries in ModifierType.
+#[doc(alias = "GDK_MODIFIER_MASK")]
+pub const MODIFIER_MASK: u32 = ffi::GDK_MODIFIER_MASK as u32;
+
+// Defines all possible DND actions. This can be used in gdk::Drop::status messages when any drop can be accepted or a more specific drop method is not yet known.
+#[doc(alias = "GDK_ACTION_ALL")]
+pub const ACTION_ALL: u32 = ffi::GDK_ACTION_ALL as u32;
+
+// Represents the current time, and can be used anywhere a time is expected.
+#[doc(alias = "GDK_CURRENT_TIME")]
+pub const CURRENT_TIME: u32 = ffi::GDK_CURRENT_TIME as u32;
+
+/// The primary button. This is typically the left mouse button, or the right button in a left-handed setup.
+#[doc(alias = "GDK_BUTTON_PRIMARY")]
+pub const BUTTON_PRIMARY: u32 = ffi::GDK_BUTTON_PRIMARY as u32;
+
+/// The middle button.
+#[doc(alias = "GDK_BUTTON_MIDDLE")]
+pub const BUTTON_MIDDLE: u32 = ffi::GDK_BUTTON_MIDDLE as u32;
+
+/// The secondary button. This is typically the right mouse button, or the left button in a left-handed setup.
+#[doc(alias = "GDK_BUTTON_SECONDARY")]
+pub const BUTTON_SECONDARY: u32 = ffi::GDK_BUTTON_SECONDARY as u32;
+
+// Used as the return value for stopping the propagation of an event handler.
+#[doc(alias = "GDK_EVENT_STOP")]
+pub const EVENT_STOP: u32 = ffi::GDK_EVENT_STOP as u32;
+
+// Used as the return value for continuing the propagation of an event handler.
+#[doc(alias = "GDK_EVENT_PROPAGATE")]
+pub const EVENT_PROPAGATE: u32 = ffi::GDK_EVENT_PROPAGATE as u32;
```

Comment:
This should probably be handled like `glib::Continue`

Reply:
Yes, definitely. There's an open issue to properly handle this one.